examples/fluids/advection.h

```diff
@@ -145,7 +146,7 @@ static inline int Exact_Advection(CeedInt dim, CeedScalar time,
 }
 
 // *****************************************************************************
-// Initial conditions for 3D advection
+// This QFunction sets the initial conditions for 3D advection
```

Comment:
This change is fine - I just don't want to see us do another massive code sweep to change all comments to sentences

Reply:
I just wanted to be consistent with line 174 below

Reply:
Like I said, I think that it looks fine here. I was just voicing a concern - specifically that I don't think we need to make a big code sweep to turn all comments into sentences.